### PR TITLE
Add No-Op open tracing implementation

### PR DESCRIPTION
**Pull request by sbly**
This is a No-Op Open Tracing implementation that I've been using for testing

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -157,6 +157,18 @@ lazy val log = project
     )
   )
 
+lazy val noop = project
```
Comment (tpolecat) [inline build.sbt:160]: Can you add this to the `dependsOn` and `aggregate` for the root project?

**Comment by sbly** [comment]
@tpolecat Thanks for reviewing - I made the requested change

**Comment by tpolecat** [comment]
Please run `sbt headerCreate` to add file headers, then I think you're all set.

**Comment by sbly** [comment]
Done!

**Comment by tpolecat** [comment]
Thanks!

**Pull request merged**